src/main/java/ts/parking/service/CashCalculations.java

```diff
@@ -0,0 +1,78 @@
+package ts.parking.service;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+
+import ts.parking.model.Currency;
+import ts.parking.model.DriverType;
+import ts.parking.model.Parking;
+
+public class CashCalculations {
+
+	public static BigDecimal calculateParkingPaymentAmount(LocalDateTime start, LocalDateTime end, DriverType driverType, Currency currency) {
+		BigDecimal paymentAmount = BigDecimal.ZERO;
+		int parkingTimeInMinutes = Math.toIntExact(ChronoUnit.MINUTES.between(start, end));
+		int parkingTimeEachStartedHour = 0;
+		if(parkingTimeInMinutes % 60 > 0) {
+			parkingTimeEachStartedHour = parkingTimeInMinutes / 60 + 1;
+		}
+		else if(parkingTimeInMinutes % 60 < 0) {
```

Comment:
A ten if? To będzie tylko jeśli end będzie mniejsze niż start, co chyba nie jest możliwe?

Reply:
Racja, pomyliłem się tutaj. Powinno być coś takiego:
if(parkingTimeInMinutes % 60 > 0 && parkingTimeInMinutes < 60) {
	parkingTimeEachStartedHour = 1;
}
else if(parkingTimeInMinutes % 60 > 0) {
	parkingTimeEachStartedHour = parkingTimeInMinutes / 60 + 1;
}
else {
	parkingTimeEachStartedHour = parkingTimeInMinutes / 60;
}